internal/quickstart/choose_sdk.go

```diff
@@ -72,11 +82,17 @@ func (m chooseSDKModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
 		switch {
+		case msg.String() == "/":
```

Comment:
we should also add this to the help keys so it's more discoverable

Reply:
Done. I added a screenshot to see it show up only on the choose SDK page.